src/components/dataPreview/DataPreviewTable.vue

```diff
@@ -22,34 +22,44 @@
 </template>
 
 <script lang="ts">
-  import { Component, Prop, Vue } from 'vue-property-decorator';
-  import { IAttributeModule } from '@/store/attributeModules/IAttributeModule';
-  @Component
-  export default class DataPreviewTable extends Vue {
-    @Prop({ required: true })
-    public module!: IAttributeModule;
-
-    get isDataLoaded() {
-      return this.module.data !== null;
-    }
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import { IAttributeModule } from '@/store/attributeModules/IAttributeModule';
+@Component
+export default class DataPreviewTable extends Vue {
+  @Prop({ required: true })
+  public module!: IAttributeModule;
 
-    get hasData() {
-      return this.module.previewData.length > 0;
-    }
+  get isDataLoaded() {
+    return this.module.data !== null;
+  }
 
-    get columnNames() {
-      return Object.keys(this.module.previewData[0]);
-    }
+  get hasData() {
+    return this.module.previewData.length > 0;
+  }
+
+  get columnNames() {
+    return Object.keys(this.module.previewData[0]);
+  }
+
+  get encodedFileUri() {
+    let csvContent = '';
+    csvContent += this.columnNames.join(';') + ';\r\n';
+
+    this.module.data!.forEach((row) => {
+      const values = Object.keys(row).map((key) => (row as any)[key]);
+      csvContent += values.join(';') + ';\r\n';
+    });
 
-    get encodedFileUri() {
-      let csvContent = 'data:text/csv;charset=utf-8,\uFEFF';
-      csvContent += this.columnNames.join(';') + ';\r\n';
+    // https://stackoverflow.com/questions/23301467/javascript-exporting-large-text-csv-file-crashes-google-chrome
+    const csvData = new Blob([csvContent], {
+      type: 'text/csv;charset=utf-8;'
+    });
 
-      this.module.data!.forEach((row) => {
-        const values = Object.keys(row).map((key) => (row as any)[key]);
-        csvContent += values.join(';') + ';\r\n';
-      });
-      return encodeURI(csvContent);
+    if (window.navigator.msSaveBlob) {
+      navigator.msSaveBlob(csvData, this.module.name + '.csv');
```

Comment:
Mä en oo varma tarvitaanko tätä. Uusimmalla Edgellä toimii ilmankin, mutta sit oli kyllä tällanen:
https://stackoverflow.com/questions/46237494/dataattachment-csv-download-doesnt-work-in-edge